nip47/event_handler.go

```diff
@@ -102,7 +114,7 @@ func (svc *nip47Service) HandleEvent(ctx context.Context, relay nostrmodels.Rela
 				Message: "The public key does not have a wallet connected.",
 			},
 		}
-		resp, err := svc.CreateResponse(event, nip47Response, nostr.Tags{}, ss)
+		resp, err := svc.CreateResponse(event, nip47Response, nostr.Tags{}, ss, svc.keys.GetNostrSecretKey())
```

Comment:
I think this is wrong. @frnandu I don't think we should reply here actually. What do you think?

Reply:
Let's just log an error and return?

Reply:
You're right, if there is no app with the key, we can't even encrypt the response, so just log the error.